sourcegraph/about
# New blog components for 4.0 launch

Closes #5729

### Changelog
- New blog layout components: `Badge` and link icon option for `Figure`
- Makes `light-gray` the default color to `Badge` to ease the use in blog
- Blog starter pack documentation update
- Updates Handbook blog component outline: PR [#4845](https://github.com/sourcegraph/handbook/pull/4845)

### Note
- Let me know if you'd like any language changed on the documentation!
<img width="756" alt="image" src="https://user-images.githubusercontent.com/59381432/190869954-3bf2827f-482d-4757-9cf2-201c38e79c0e.png">

### Test
1. Ensure prettier has standardized the proposed changes.
2. View blog starter pack documentation at `/blog/starter-pack`
